samples/apply-dictionary-renderer-to-graphics-overlay/README.metadata.json

```
@@ -0,0 +1,33 @@
+{
+    "category": "Scenes",
```

Category change required: 


```suggestion
    "category": "Visualization",
```